.ci_support/osx64.yaml

```diff
@@ -1,8 +1,12 @@
-c_compiler:
-  - clang
-cxx_compiler:
-  - clangxx
-fortran_compiler:
-  - gfortran
+MACOSX_DEPLOYMENT_TARGET:
```

Comment:
Can you revert the changes to this file (after addressing any other comments so we still see it's green). The CI will fail but I'll merge regardless (the `recipes/tilt/conda_build_config.yaml` will make sure the feedstock is correct).